client/src/assets/stylesheets/pages/LandingPage.scss

```diff
@@ -26,8 +58,42 @@
 	&__pagesection{
 		padding: 50px 0 !important;
 	}
+	// &__lab_title{
```

Comment:
It looks like this code is commented out, so this could probably be deleted (unless it's actually being used somewhere)